docs/workflow/development.md

```diff
@@ -1,84 +1,113 @@
+-   No assignee: _Open_, _Done_
+-   Assignee works on issue/pull request (if more or less progress happens on it): _In Progress_
+-   Assignee is responsible to bring a pull request to `main` (e.g. reviewer, tester): _Request for Review_
 
-#### Terms
+!!! note
 
-**Assignee:** The person that is responsible for a card (has the **exclusive** right to move a card(!) as soon as there is one). The assignee's job depends on the Kanban column:
+    Github distinguishes between _reviewers_ and _assignees_.
+    We also do that.
+    While the _assignee_ (in most cases one person) is responsible for the card and bring the branch into the main, the _reviewer(s)_ (can be more than one) can review the code at any time in addition to the notes that were given by the _assignee_.
+    In general: The more reviewers the better the resulting code (That being said: feel free to review code).
 
-- No assignee: *Open*, *Done*
-- Assignee works on issue/pull request (if more or less progress happens on it): *In Progress*
-- Assignee is responsible to bring a pull request to `main` (e.g. reviewer, tester): *Request for Review*
+**Author:** The person that created this card (issue/pull request).
+Responsible for answering questions (issues/pull requests) and implement/discuss requested changes (pull requests).
 
-**Note** that github distinguishes between *reviewers* and *assignees*. We also do that. Kind of. While the *assignee* (in most cases one person) is responsible for the card and bring the branch into the main, the *reviewer(s)* (can be more than one) can review the code at any time in addition to the notes that were given by the *assignee*. In general: The more reviewers the better the resulting code (That being said: feel free to review code).
+### Columns
 
-| *Open* | *In Progress* | *Request for Review* | *Done* |
-|--------|---------------|----------------------|--------|
-|--------| Developer     | Reviewer             |--------|
+Our project board is divided into the following four columns:
 
-**Author:** The person that created this card (issue/pull request). Responsible for answering questions (issues/pull requests) and implement/discuss requested changes (pull requests).
+| _Open_   | _In Progress_            | _Request for Review_    | _Done_   |
+| -------- | ------------------------ | ----------------------- | -------- |
+| -------- | Developer is responsible | Reviewer is responsible | -------- |
+
+??? question "The _hidden_ column"
+
+    The _hidden_ column is a special column that is not visible on the project board.
+    It contains all open issues, that are not in _Open_ and that currently should not be worked on.
+    This is to keep the project board clean and focused on the current iteration.
```

Comment:
Does this need to be here? I don't think this is relevant for most people.

Reply:
That's why I its hidden using the `???`
I think it still might be interesting, to clarify why somebody would move an issue there

Reply:
Ok, fair, but I don't like the intense color of that admonition box. Its easily the most attention drawing item on the page.

Reply:
Okay, I changed it to info, do you prefer it like that?